### ISSUE TITLE
测试精度问题

### ISSUE THREAD

**Issue by wangyuxin87** [Shun14/TextBoxes_plusplus_Tensorflow#6]
作者你好 我想问下这个tensorflow版本能达到原文精度吗？ 略次还是略高？

**Comment by wangyuxin87** [comment]
期待你的回复！

**Comment by basaltzhang** [comment]
I use the released ckpt in this git, test on the ICDAR Incidental 2015 test set. The result seems not good enough.
Recall/Precision/Hmean: 0.666/0.035/0.067
Using prob score, under the best hmean,  p/r/f1-score: 0.453/0.406/0.428
I assume the author didn't train on this training set, so next i'll try and show the results.

**Comment by Shun14** [comment]
> I use the released ckpt in this git, test on the ICDAR Incidental 2015 test set. The result seems not good enough.
> Recall/Precision/Hmean: 0.666/0.035/0.067
> Using prob score, under the best hmean, p/r/f1-score: 0.453/0.406/0.428
> I assume the author didn't train on this training set, so next i'll try and show the results.

The released model was trained on SynthText Dataset.You can train your own model base on it.

**Comment by Shun14** [comment]
> 期待你的回复！

大概低一点，因为有些论文里提到的trick我没加，比如数据增强的时候，对小物体的crop的时候的规则

**Comment by wangyuxin87** [comment]
好的 谢谢！

**Comment by basaltzhang** [comment]
The author says N in loss function is the number of default boxes that match groundtruth boxes, but in 
 this code is the batch size instead.

**Comment by basaltzhang** [comment]
Follow the author's training parameters in Table, I have the following result:
p/r/f1-score: 0.773/0.697/0.733

**Comment by qingfengyy** [comment]
> Follow the author's training parameters in Table, I have the following result:
> p/r/f1-score: 0.773/0.697/0.733

Could you tell me How  you get this result, have you modified the loss function or other ways.The loss value does not decreas when training on icdar2015.

**Comment by jercas** [comment]
> > Follow the author's training parameters in Table, I have the following result:
> > p/r/f1-score: 0.773/0.697/0.733
> 
> Could you tell me How you get this result, have you modified the loss function or other ways.The loss value does not decreas when training on icdar2015.

I have the some problem when i use the released model which pre-trained on Synthtext, the loss seem ridiculous and wave between 70 to 200 (sometime even higher). Have you guys solved this problem?
